packages/edit-site/src/components/post-list/index.js

```diff
@@ -142,15 +142,15 @@ export default function PostList( { postType } ) {
 	const history = useHistory();
 	const location = useLocation();
 	const { postId, quickEdit = false } = location.params;
-	const [ selection, setSelection ] = useState( [ postId ] );
+	const [ selection, setSelection ] = useState( postId?.split( ',' ) ?? [] );
 	const onChangeSelection = useCallback(
 		( items ) => {
 			setSelection( items );
 			const { params } = history.getLocationWithParams();
 			if ( ( params.isCustom ?? 'false' ) === 'false' ) {
 				history.push( {
 					...params,
-					postId: items.length === 1 ? items[ 0 ] : undefined,
+					postId: items.join( ',' ),
```

Comment:
I noticed that his change was triggering 404 requests in the Pages table view. My proposed fix - #64254.